api/address/routes.go

```diff
@@ -459,6 +462,49 @@ func GetESDTTokensWithRole(c *gin.Context) {
 	)
 }
 
+// GetOwnedNFTs returns the token identifiers of the tokens where a given address is the owner
+func GetOwnedNFTs(c *gin.Context) {
```

Comment:
the naming is wrong. GetNFTTokenIDsRegisteredByAddress

Reply:
done